### DISCUSSION TITLE
How to edit the Time label?

### DISCUSSION THREAD

**Discussion by jonathangoncalves** [vkurko/calendar#199] [Q&A] [answered]
I want to put some extra info in the event "time" label but I can't find a way to do it. I tried to use [slotLabelFormat](https://github.com/vkurko/calendar#slotlabelformat) and [eventTimeFormat](https://github.com/vkurko/calendar#eventtimeformat) but had no difference. Is it possible and how to do it?

Example: Add a text or html after the time
Time: 10:00 AM - some text or html
Title: Some Meeting

**Comment by vkurko** [comment, marked as answer]
This should output TEXT instead of time:
```js
eventTimeFormat: function (info) {
    return 'TEXT';
},
```

But in this case you need to format the time yourself from `info.start` and `info.end`.
